File: services/src/main/java/com/liferay/mobile/sdk/v7/layoutsetbranch/LayoutSetBranchService.java
package com.liferay.mobile.sdk.v7.layoutsetbranch;

import com.liferay.mobile.sdk.Call;
import com.liferay.mobile.sdk.annotation.Param;
import com.liferay.mobile.sdk.annotation.Path;
import com.liferay.mobile.sdk.http.Response;
import java.lang.String;
import org.json.JSONArray;
import org.json.JSONObject;

@Path("/layoutsetbranch")
public interface LayoutSetBranchService {
  @Path("/add-layout-set-branch")
  Call<JSONObject> addLayoutSetBranch(@Param(name = "groupId") long groupId, @Param(name = "privateLayout") boolean privateLayout, @Param(name = "name") String name, @Param(name = "description") String description, @Param(name = "master") boolean master, @Param(name = "copyLayoutSetBranchId") long copyLayoutSetBranchId, @Param(name = "serviceContext", className = "") JSONObject serviceContext);

  @Path("/delete-layout-set-branch")
  Call<Response> deleteLayoutSetBranch(@Param(name = "layoutSetBranchId") long layoutSetBranchId);

  @Path("/get-layout-set-branches")
  Call<JSONArray> getLayoutSetBranches(@Param(name = "groupId") long groupId, @Param(name = "privateLayout") boolean privateLayout);

  @Path("/merge-layout-set-branch")
  Call<JSONObject> mergeLayoutSetBranch(@Param(name = "layoutSetBranchId") long layoutSetBranchId, @Param(name = "mergeLayoutSetBranchId") long mergeLayoutSetBranchId, @Param(name = "serviceContext", className = "") JSONObject serviceContext);

  @Path("/update-layout-set-branch")
  Call<JSONObject> updateLayoutSetBranch(@Param(name = "groupId") long groupId, @Param(name = "layoutSetBranchId") long layoutSetBranchId, @Param(name = "name") String name, @Param(name = "description") String description, @Param(name = "serviceContext", className = "") JSONObject serviceContext);
}

File: windows/Liferay.SDK/Service/V62/LayoutSetBranch/LayoutSetBranchService.cs
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace Liferay.SDK.Service.V62.LayoutSetBranch
{
	public class LayoutSetBranchService : ServiceBase
	{
		public LayoutSetBranchService(ISession session)
			: base(session)
		{
		}

		public async Task<IDictionary<string, object>> AddLayoutSetBranchAsync(long groupId, bool privateLayout, string name, string description, bool master, long copyLayoutSetBranchId, IDictionary<string, object> serviceContext)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("privateLayout", privateLayout);
			_parameters.Add("name", name);
			_parameters.Add("description", description);
			_parameters.Add("master", master);
			_parameters.Add("copyLayoutSetBranchId", copyLayoutSetBranchId);
			_parameters.Add("serviceContext", serviceContext);

			var _command = new JsonObject()
			{
				{ "/layoutsetbranch/add-layout-set-branch", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (IDictionary<string, object>)_obj;
		}

		public async Task DeleteLayoutSetBranchAsync(long layoutSetBranchId)
		{
			var _parameters = new JsonObject();

			_parameters.Add("layoutSetBranchId", layoutSetBranchId);

			var _command = new JsonObject()
			{
				{ "/layoutsetbranch/delete-layout-set-branch", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}

		public async Task<IDictionary<string, object>> GetLayoutSetBranchesAsync(long groupId, bool privateLayout)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("privateLayout", privateLayout);

			var _command = new JsonObject()
			{
				{ "/layoutsetbranch/get-layout-set-branches", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (IDictionary<string, object>)_obj;
		}

		public async Task<IDictionary<string, object>> MergeLayoutSetBranchAsync(long layoutSetBranchId, long mergeLayoutSetBranchId, IDictionary<string, object> serviceContext)
		{
			var _parameters = new JsonObject();

			_parameters.Add("layoutSetBranchId", layoutSetBranchId);
			_parameters.Add("mergeLayoutSetBranchId", mergeLayoutSetBranchId);
			_parameters.Add("serviceContext", serviceContext);

			var _command = new JsonObject()
			{
				{ "/layoutsetbranch/merge-layout-set-branch", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (IDictionary<string, object>)_obj;
		}

		public async Task<IDictionary<string, object>> UpdateLayoutSetBranchAsync(long groupId, long layoutSetBranchId, string name, string description, IDictionary<string, object> serviceContext)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("layoutSetBranchId", layoutSetBranchId);
			_parameters.Add("name", name);
			_parameters.Add("description", description);
			_parameters.Add("serviceContext", serviceContext);

			var _command = new JsonObject()
			{
				{ "/layoutsetbranch/update-layout-set-branch", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (IDictionary<string, object>)_obj;
		}
	}
}